src/Sylius/Behat/Context/Setup/CliContext.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Setup;
+
+use Behat\Behat\Context\Context;
+use Sylius\Bundle\InstallerBundle\Command\InstallSampleDataCommand;
+use Sylius\Bundle\InstallerBundle\Command\SetupCommand;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Helper\DialogHelper;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+/**
+ * @author Magdalena Banasiak <magdalena.banasiak@lakion.com>
+ */
+final class CliContext implements Context
+{
+    /**
+     * @var KernelInterface
+     */
+    private $kernel;
+
+    /**
+     * @var Application
+     */
+    private $application;
+
+    /**
+     * @var CommandTester
+     */
+    private $tester;
+
+    /**
+     * @var DialogHelper
+     */
+    private $dialog;
+
+    /**
+     * @var SetupCommand
+     */
+    private $command;
+
+    /**
+     * @var array
+     */
+    private $inputChoices = [
+        'currency' => '',
+        'name' => ' Name',
+        'surname' => ' Surname',
+        'e-mail' => ' test@email.com',
+        'password' => ' pswd',
+        'confirmation' => ' pswd',
+     ];
+
+    /**
+     * @param KernelInterface $kernel
+     */
+    public function __construct(KernelInterface $kernel)
+    {
+        $this->kernel = $kernel;
```

Comment:
If we rename this context. We can setup whole background i mean this app/command/tester creation here.
Then in steps `iRun....` simply execute proper command.